lib/WP_Auth0_DBManager.php

```diff
@@ -170,6 +170,19 @@ public function install_db( $version_to_install = null ) {
 			unset( $update_options['auth0js-cdn'] );
 			unset( $update_options['passwordless_cdn_url'] );
 			unset( $update_options['cdn_url_legacy'] );
+
+			unset( $update_options['social_twitter_key'] );
+			unset( $update_options['social_twitter_secret'] );
+			unset( $update_options['social_facebook_key'] );
+			unset( $update_options['social_facebook_secret'] );
+			unset( $update_options['connections'] );
```

Comment:
this is the only line not repeating in the other diff. Is this intentional?

Reply:
@lbalmaceda - Yes, old sub-setting array that was removed